x-pack/test/cloud_security_posture_functional/pages/cis_integration.ts

```diff
@@ -41,6 +42,87 @@ export default function (providerContext: FtrProviderContext) {
       await kibanaServer.savedObjects.cleanStandardList();
     });
 
+    describe('CNVM AWS', () => {
+      it('Hyperlink on PostInstallation Modal should have the correct URL', async () => {
+        await cisIntegration.navigateToAddIntegrationCnvmPage();
+        await cisIntegrationGcp.clickSaveButton();
+        pageObjects.header.waitUntilLoadingHasFinished();
+        expect(
+          (await cisIntegration.getUrlOnPostInstallModal()) ===
+            'https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-whatis-howdoesitwork.html'
```

Comment:
I have mixed feelings about checking the exact url. Having it like this means that every time we change the implementation (eg. the link changes and we update it) we also need to update the test. At the same time, it doesn't give us confidence that when the link becomes broken (eg. Amazon moves the docs somewhere else) we will catch it with the test. I'd personally prefer checking that the link is just present where we want it to be present, but don't check the exact url. But we can live with this implementation for some time and see if it causes us problems